src/main/java/fixie/dictionary_service/service/IDictionaryService.java

```diff
@@ -0,0 +1,24 @@
+package fixie.dictionary_service.service;
+
+import fixie.dictionary_service.dto.ActivityDictionaryDTO;
+import fixie.dictionary_service.dto.PartTypeDTO;
+import fixie.dictionary_service.entity.ActivityDictionary;
+import fixie.dictionary_service.entity.PartType;
+
+
```

Comment:
unnecessary blank lines, imports can be next to each other. You can make 1 new line before IDictionaryService declaration.

Reply:
fixed